qsh/__init__.py

```diff
@@ -191,7 +191,7 @@ def __init__(self, filename=None, mode=None):
         self.fileobj = gzip.open(filename, mode)
 
         signature = self.read(len(self.header.signature.encode("utf8")))
-    
+
```

Comment:
All those changes were made by Atom. I think it's OK

Reply:
Yeah it's OK!